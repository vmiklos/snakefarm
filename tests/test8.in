debug 1
out 1
time 100
load test8.map
addplayer 1
addsnake 1 1 1 1 1 1
12 0
13 0
20 0
19 0
18 0
endsnake
step
comment # a jatekos kigyoi kozt kell lennie egy 2-es sorszamunak
showplayer 1
exit