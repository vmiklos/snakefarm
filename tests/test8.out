Event MapLoad test8.map
Event Step 1
StepEvent Snake_Snake 1
StepEvent SnakeSplit 4 2
End Step
# a jatekos kigyoi kozt kell lennie egy 2-es sorszamunak
Player 1
1 3 1 1 1
2 1 1 1 1
endplayer
Event Exit
